### PR TITLE
Sepa payment type

### PR DESCRIPTION
**Pull request by maria-mcparland**
- Added in SEPA payments (currently won't work as the accounts are not ready but can see mocked flow)
- Removed validation on the payment form as we want to see negative scenarios from API
- Extracted out components of the payment form to custom select and input fields

### CODE REVIEW

File: app/client/src/components/sendPayments/FormFields/SelectField.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+};
+
+export default function SelectField({
+  id, options, register, label,
+}: SelectFieldProps) {
+  let defaultValue;
+  if (id === 'creditorAccount') {
+    defaultValue = JSON.stringify(options[1]);
+  }
+  const renderOption = (option: string | AccountType) => {
```
Comment (KyleJohnst) [inline app/client/src/components/sendPayments/FormFields/SelectField.tsx:21]: Would be good to start incorporating the return of functions going forward.

File: app/client/src/types/globalPaymentApiTypes.ts [hunk 2]
```diff
@@ -29,17 +30,25 @@ export type PaymentsResponse = {
 };
 
 type AgentType = {
-  financialInstitutionId: {
-    clearingSystemId: {
-      id: string,
-      idType?: string
+  financialInstitutionId:
+  (
+    { bic : 'CHASGB2L' | 'CHASINBX' | 'CHASAU2X' | 'CHASSGSG' | 'CHASUS33' | 'CHASMYKX' | 'CHASHKHH' | 'CHASBRSP' | 'CHASDEFX' |
```
Comment (KyleJohnst) [inline app/client/src/types/globalPaymentApiTypes.ts:35]: Looks like some repetition here and in line 19 of accountTypes.ts

**Review by KyleJohnst — APPROVED** [review]
Looks good

File: app/client/src/types/globalPaymentApiTypes.ts (hunk 2 above)
Comment (maria-mcparland) [reply app/client/src/types/globalPaymentApiTypes.ts:35]: Good spot, need to refine all these types. Coming soon....in next PR

**Pull request merged**